modules/experimental-layers/src/bitmap-layer/bitmap-layer-fragment.js

```diff
@@ -0,0 +1,79 @@
+export default `
+#define SHADER_NAME bitmap-layer-fragment-shader
+
+#ifdef GL_ES
+precision highp float;
+#endif
+
+uniform sampler2D uBitmap0;
+uniform sampler2D uBitmap1;
+uniform sampler2D uBitmap2;
+uniform sampler2D uBitmap3;
+uniform sampler2D uBitmap4;
+uniform sampler2D uBitmap5;
+uniform sampler2D uBitmap6;
+uniform sampler2D uBitmap7;
+uniform sampler2D uBitmap8;
+uniform sampler2D uBitmap9;
```

Comment:
Most systems support 16 textures but there are systems that only support 8 (GL.MAX_TEXTURE_IMAGE_UNITS)[http://webglstats.com/webgl/parameter/MAX_TEXTURE_IMAGE_UNITS]

Should we use one of these values as limit? 8 would be the safest.

It seems to be possible to declare uniforms to be arrays of samplers, but not sure how well luma.gl supports it:

https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/conformance/glsl/bugs/sampler-array-using-loop-index.html

Future idea: On WebGL2 one can use `Texture2DArray` that support a minimum of `256` indices.